### PR TITLE
linux: build musl/libunwind for i686

### PR DESCRIPTION
**Pull request by japaric**
Now we can use the "linux" docker image to build libstd for i686-unknown-linux-musl with the
following command:

configure --disable-rustbuild --musl-root=/musl-i686 --target=i686-unknown-linux-musl && make

r? @alexcrichton

### CODE REVIEW

File: slaves/linux/build-musl.sh [hunk 1]
```diff
@@ -7,9 +7,15 @@ export CFLAGS=-fPIC
 # Support building MUSL
 curl http://www.musl-libc.org/releases/musl-1.1.11.tar.gz | tar xzf -
 cd musl-1.1.11
+# for x86_64
 ./configure --prefix=/musl --disable-shared
```
Comment (alexcrichton) [inline slaves/linux/build-musl.sh]: Could you change this to install to `/musl-x86_64`?

**Comment by alexcrichton** [comment]
Nice! I think we may want to change this at some point so you can configure the i686 and x86_64 musl target at the same time, but right now they both try to use `musl-gcc` which fails. Something we can fix later though!

**Comment by japaric** [comment]
> Could you change this to install to /musl-x86_64?

Done. Don't forget to update the `--musl-root` flag on the rustbuild side!

**Comment by alexcrichton** [comment]
pffffft thinking we don't test changes before we merge them

I will indeed attempt to remember :)

**Pull request merged**